frontend/app/src/components/settings/data-security/DataManagement.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <v-row no-gutters>
+    <v-col>
+      <v-card>
+        <v-card-title>Manage Data</v-card-title>
+        <v-card-subtitle>
+          Allows you to purge data cached by rotki
+        </v-card-subtitle>
+        <v-form ref="form">
+          <v-card-text>
+            <h3>Exchange Data</h3>
+            <v-row align="center">
+              <v-col>
+                <v-select
+                  v-model="exchange"
+                  class="data-management__fields__exchange"
+                  :items="exchanges"
+                  :success-messages="
+                    exchangePurgeSuccess
+                      ? ['Cached data where successfully purged']
```

Comment:
where -> were. Also would it be possible to indicate which cached data these were?